apps/photos/src/components/PhotoFrame.tsx

```diff
@@ -82,6 +103,148 @@ const PhotoFrame = ({
     const [isSourceLoaded, setIsSourceLoaded] = useState(false);
     const [conversionFailed, setConversionFailed] = useState(false);
 
+    const avatarColors: string[] = [
+        '#76549A',
+        '#DF7861',
+        '#94B49F',
+        '#87A2FB',
+        '#C689C6',
+        '#937DC2',
+        '#325288',
+        '#85B4E0',
+        '#C1A3A3',
+        '#E1A059',
+        '#426165',
+        '#6B77B2',
+        '#957FEF',
+        '#DD9DE2',
+        '#82AB8B',
+        '#9BBBE8',
+        '#8FBEBE',
+        '#8AC3A1',
+        '#A8B0F2',
+        '#B0C695',
+        '#E99AAD',
+        '#D18484',
+        '#78B5A7',
+    ];
+
+    useEffect(() => {
+        const fetchOwnerEmails = async () => {
+            try {
+                const collection: CollectionItem[] =
+                    await getLocalCollections();
```

Comment:
We can define a method inside collectionSerive that will return userID to email Map. We need the id to email information for sharees as well. In a shared collection, we can have multiple people who might have added photos in a colleciton.